wordpress-mobile/WordPress-Android
# Fix "No Activity found to handle VIEW" crash

Fixes #4899 by checking there is a default app installed on the device that is able to open the passed Uri.

To test on Emulator: 
- Open settings and disable the Browser.
- Start WP and tap on view site

A toast should be shown on the screen with the correct error message in it.


On device you need to disable any installed browsers, and maybe disable the system webview.
